pkgs/top-level/all-packages.nix

```diff
@@ -22950,7 +22950,9 @@ with pkgs;
 
   govendor = callPackage ../development/tools/govendor { };
 
-  go-tools = callPackage ../development/tools/go-tools { };
+  go-tools = callPackage ../development/tools/go-tools {
+    buildGoModule = buildGo117Module;
+  };
```

Comment:
Please revert this. Go 1.17 is still the default.

Reply:
This build fails with `//go:build comment without // +build` error when compiling `go/ir/builder.go`. This error only shows up on Go 1.16, because they were transitioning between the `// +build` comments and `//go:build` comments. To me, this appears as if `buildGoModule` uses Go 1.16 by default. And since trying to use `buildGo118Module` doesn't show up here, I decided to use a version that still allowed it to build.

Reply:
Gotcha, I had an older version of master, which is why it wasn't working locally.

Reply:
Updated